web/ui/static/js/graph.js

```diff
@@ -518,7 +519,6 @@ Prometheus.Graph.prototype.transformData = function(json) {
       color: palette.color()
     };
   });
-  Rickshaw.Series.zeroFill(data);
```

Comment:
You can use `Rickshaw.Series.fill(data, null)` instead of re-implementing it.

https://github.com/shutterstock/rickshaw/blob/master/src/js/Rickshaw.Series.js#L137-L139